spec/models/user_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User do
+  # rubocop:disable RSpec/ExampleLength
+  it 'populates a User object' do
+    # rubocop:disable Layout/LineLength
+    dataverse_user_group = 'cn=edu:berkeley:app:calnet-spa:group-spa-ucblibdataverse,ou=campus groups,dc=berkeley,dc=edu'
```

Comment:
is it sufficient just to call `described_class::DEOVOLENTE_USER_GROUP` below instead? and if so, do we need another example to check for the presence of the constant?

Reply:
Just re-read this one and implemented that. TBH I didn't realize you could do that! VERY SLICK!